app/code/community/Adyen/Payment/Model/Adyen/Cc.php

```diff
@@ -92,16 +97,17 @@ public function validate()
         parent::validate();
     }
 
-    public function getPossibleInstallments() {
+    public function getPossibleInstallments()
+    {
         // retrieving quote
-        $quote = (Mage::getModel('checkout/type_onepage') !== false)? Mage::getModel('checkout/type_onepage')->getQuote(): Mage::getModel('checkout/session')->getQuote();
+        $quote = (Mage::getModel('checkout/type_onepage') !== false) ? Mage::getModel('checkout/type_onepage')->getQuote() : Mage::getModel('checkout/session')->getQuote();
 
         // get selected payment method for now
         $payment = $quote->getPayment();
 
         $ccType = null;
-        if($payment && !empty($payment)) {
-            if($payment->getMethod()) {
+        if ($payment && !empty($payment)) {
+            if ($payment->getMethod()) {
```

Comment:
no necessary if - I think something like this should work
```suggestion
            if (!empty($payment) && !empty($payment)) {
```
Also the check if the $payment is true has been performed in the empty function

(For the 109 and 110 line)